### DISCUSSION TITLE
can snap-it work with oracle database instead of mysql

### DISCUSSION THREAD

**Discussion by ftjn993** [snipe/snipe-it#13097] [Q&A]
we are going to install the latest version of snap-it, while we have no capacity to support mysql, at the same time, we have oracle 19c database license, in this case, we wonder if snap-it can work with oracle database without any special change to the coding/installation step/configuration files?   thank you

**Comment by marcusmoore** [comment]
It might be possible but would probably require some additional configuration. You might be interested in this [package](https://github.com/yajra/laravel-oci8).

Unfortunately we don't have the ability to test it.
